### PR TITLE
Fix behats

### PR DESCRIPTION
**Pull request by phaseinducer**
Fix for :
features/category/edit_a_category.feature:10
features/product/toggle_status_product.feature:12

### CODE REVIEW

**Comment by jmleroux** [comment]
GTM

File: features/category/edit_a_category.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@javascript
```
Comment (skeleton) [inline features/category/edit_a_category.feature]: why put this tag here ? Julien spent some time to remove it and add it to each necessary scenario

**Comment by skeleton** [comment]
1 question, then GTM

File: features/category/edit_a_category.feature (hunk 1 above)
Comment (phaseinducer) [inline features/category/edit_a_category.feature]: IRL

**Pull request merged**